src/renderer/components/team/dialogs/TaskCommentsSection.tsx

```
@@ -19,14 +19,29 @@ import {
   Eye,
+  Loader2,
   MessageSquare,
   Reply,
   Send,
   X,
 } from 'lucide-react';
 
 import type { MentionSuggestion } from '@renderer/types/mention';
-import type { ResolvedTeamMember, TaskComment } from '@shared/types';
+import type {
+  AttachmentMediaType,
```

ESLint

[error] 'AttachmentMediaType' is defined but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] Remove this unused import of 'AttachmentMediaType'.

(sonarjs/unused-import)